front/Navbar.tsx

```diff
@@ -17,7 +17,7 @@ const Navbar = () => {
   const clickHandler = () => resetIndexState(setPage, setTab, loggedInUser)
   return (
     <nav className="navbar navbar-light">
-      <div className="container">
+      <div className="container" data-cy = "header">
```

Comment:
Please add a space before and after the equal sign in the data-cy attribute to keep the code consistent and clean.